ui/components/app/assets/token-cell/__snapshots__/token-cell.test.js.snap

```diff
@@ -48,14 +48,12 @@ exports[`Token Cell should match snapshot 1`] = `
             >
               TEST
             </span>
-            <div
-              class="mm-box mm-box--display-flex"
+            <p
+              class="mm-box mm-text mm-text--body-md mm-text--ellipsis mm-box--color-text-alternative"
+              data-testid="multichain-token-list-item-token-name"
             >
-              <p
-                class="mm-box mm-text mm-text--body-md mm-text--font-weight-normal mm-text--ellipsis mm-box--color-text-default"
-                data-testid="token-increase-decrease-percentage-0xAnotherToken"
-              />
-            </div>
+              TEST
+            </p>
           </div>
           <div
```

Comment:
Not sure why this snapshot is changing, but visually I don't see any difference in the UI. I suspect that the snapshot unit test may have been incorrect to begin with?

Reply:
Resolving as the snapshot is no longer changed and this comment isn't relevant anymore